setup.py

```diff
@@ -444,13 +444,17 @@ def run(self):
 # The list below uses the union of include_dirs and library_dirs for
 # reasons of simplicity.
 
+
 def prebuild_libchtslib(ext, force):
-    if HTSLIB_MODE not in ['shared', 'separate']: return
+    if HTSLIB_MODE not in ['shared', 'separate']:
+        return
     write_configvars_header("htslib/config_vars.h", ext, "HTS")
 
+
```

Comment:
For small functions, this is counterproductive.

I suggest you leave the most prescriptive number-of-blank-lines rules deactivated.